cmake/external/grpc.cmake

```diff
@@ -33,7 +33,7 @@ ExternalProject_Add(
     extern_grpc
     DEPENDS protobuf zlib
     GIT_REPOSITORY "https://github.com/grpc/grpc.git"
-    GIT_TAG "v1.10.x"
+    GIT_TAG "v1.8.x"
```

Comment:
I'm still looking into this issue https://github.com/PaddlePaddle/Paddle/issues/10153, version `1.10.x` should have some performance boost compared to `1.8.x`, if the machine you are using encounters this issue, can you please use another one for test?

Reply:
Done